src/AutoMapper/ValueTransformerConfiguration.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Linq.Expressions;
+
+namespace AutoMapper
+{
+    public class ValueTransformerConfiguration
```

Comment:
If it's a struct instead of a class `public bool IsMatch(PropertyMap propertyMap)` shouldn't exist, was my understanding of things.

Therefore you would need `public Func<PropertyMap, bool> IsMatch { get; }` because it's not a function but a reference to one.